tests/test_modeling_bart.py

```diff
@@ -492,7 +492,9 @@ def test_mnli_inference(self):
         inputs_dict = prepare_bart_inputs_dict(model.config, input_ids)
         # Test that model hasn't changed
         with torch.no_grad():
-            batched_logits, features = model(**inputs_dict)
+            outputs = model(**inputs_dict)[:]
```

Comment:
```suggestion
            outputs = model(**inputs_dict).to_tuple()
```
Might be easier toread like this?

Reply:
Actually the `[:]` part wasn't supposed to be part of this PR. The `[0]` and `[1]` right below should be enough.